apache/cloudstack
# Setting snapshot removed on timeout

## Description

When creating a snapshot times-out on the management server, the record in snapshot_store_ref table is deleted.

By setting the snapshot to "removed" on timeout, the record is not deleted.







Fixes: #3559 
## Types of changes


## Screenshots (if appropriate):

## How Has This Been Tested?




* Set the global variable: backup.snapshot.wait to 2 seconds.
* Create a volume snapshot.
* Check the snapshots table for a snapshot record and check for a corresponding record in snapshot_store_ref table.
